src/openrct2/interface/Window_internal.cpp

```diff
@@ -31,6 +32,11 @@ namespace OpenRCT2
     {
         widgets.clear();
         widgets.insert(widgets.end(), newWidgets.begin(), newWidgets.end());
+
+        // We need to compensate for the enlarged title bar for windows that do not
+        // constrain the window height between tabs (e.g. chart tabs)
+        if (!(flags & WF_NO_TITLE_BAR) && Config::Get().interface.EnlargedUi)
```

Comment:
Work out why `flags` are wrong when this function is called from a window constructor.

Reply:
`WF_NO_TITLE_BAR` was set to use bit 18, while `WindowBase::flags` is a `uint16_t`... _Yeah_, that's not going to work.